makefiles/host/rootfs.mk

```diff
@@ -57,17 +60,14 @@ build/nctrl/target/release/nctrl: build/nctrl/.copy_stamp
 	FUSE_CROSS_STATIC_LIB=fuse \
 	cargo build --release --target=armv7-unknown-linux-musleabihf
 
-OPENOCD_VERSION = d46f28c2ea2611f5fbbc679a5eed253d3dcd2fe3
-OPENOCD_SOURCE = build/openocd-$(LINUX_VERSION).git
-
 $(OPENOCD_SOURCE): build/root.fs/.base_install
 	@mkdir -p $(@D)
 	rm -rf $@
 	git clone https://repo.or.cz/openocd.git $@
 	(cd $@ && git reset --hard $(OPENOCD_VERSION))
 	touch $@/.scmversion
 
-$(OPENOCD_SOURCE)/.build_stamp: $(OPENOCD_SOURCE)
+$(OPENOCD_SOURCE)/.build_stamp: $(OPENOCD_SOURCE) build/root.fs/.base_install
 	(cd $(@D) && ./bootstrap)
 	(cd $(@D) && ./configure --target $(CROSS) --prefix ../root.fs/ --enable-sysfsgpio )
```

Comment:
`configure: error: expected an absolute directory name for --prefix: ../root.fs`